fontir/src/coords.rs

```diff
@@ -29,11 +29,17 @@ pub struct UserCoord(OrderedFloat<f32>);
 #[derive(Serialize, Deserialize, Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct NormalizedCoord(OrderedFloat<f32>);
 
+/// A set of per-axis coordinates that define a specific location in a coordinate system.
+///
+/// E.g. a user location is a `Location<UserCoord>`.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
+pub struct Location<T>(pub BTreeMap<String, T>);
```

Comment:
my personal feeling with these sorts of things is that only very rarely should the inner type be `pub`; it makes it easy to be undisciplined in the future. I would generally prefer to just add little wrapper methods for things like `insert` and `contains`, as you need them.

Reply:
Also if the key here is always a valid opentype tag, I would encourage you to use the `Tag` type from font-types!

Reply:
It's actually a *name* ... I suppose the non-obviousness of that suggests I should make name a type

Reply:
(sigh) that and the fact I called it tag sometimes. Maybe I jet lagged myself into confusion...